docs/data-sources/equinix_fabric_routingprotocol.md

```diff
@@ -0,0 +1,165 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "equinix_fabric_routingprotocol Data Source - terraform-provider-equinix"
+subcategory: "Fabric"
+description: |-
+  Fabric V4 API compatible data resource that allow user to fetch routing protocol for a given UUID
+  ~> Note Equinix Fabric v4 resources and datasources are currently in Beta. The interfaces related to equinix_fabric_ resources and datasources may change ahead of general availability. Please, do not hesitate to report any problems that you experience by opening a new issue https://github.com/equinix/terraform-provider-equinix/issues/new?template=bug.md
+---
+
+# equinix_fabric_routingprotocol (Data Source)
+
+Fabric V4 API compatible data resource that allow user to fetch routing protocol for a given UUID
+
+~> **Note** Equinix Fabric v4 resources and datasources are currently in Beta. The interfaces related to `equinix_fabric_` resources and datasources may change ahead of general availability. Please, do not hesitate to report any problems that you experience by opening a new [issue](https://github.com/equinix/terraform-provider-equinix/issues/new?template=bug.md)
```

Comment:
Is this still the case? When can we call the Equinix Fabric API, resources, and datasources stable?

Reply:
The Fabric Gateway(which is being added here) product is still under development and a beta release will be available soon. We can mark it stable in the next few months or whenever we go GA with this product.